packages/elements-core/src/components/TryIt/TryIt.tsx

```diff
@@ -259,7 +259,7 @@ export const TryIt: React.FC<TryItProps> = ({
 
       {formDataState.isFormDataBody ? (
         <FormDataBody
-          specification={formDataState.bodySpecification}
+          mediaTypeContent={formDataState.bodySpecification}
```

Comment:
is `FormDataBody` a publicly exported type? if so this is a breaking change...

Reply:
We export only `TryIt` and `TryItWithRequestSamples` so that should be safe

Reply:
un-renamed in 934903fa